src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -100,6 +106,10 @@ @implementation GRPCCall {
   GRPCCall *_retainSelf;
 
   GRPCRequestHeaders *_requestHeaders;
+
+  BOOL _unaryCall;
+
+  NSMutableArray *_unaryOpBatch;
```

Comment:
Let's put these two ivars in consecutive lines, and add a comment explaining their purpose.